File: app/src/main/java/com/muthu/pdfview/PDFViewActivity.java
package com.muthu.pdfview;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.webkit.WebSettings;
import android.webkit.WebView;

@SuppressLint("SetJavaScriptEnabled")
public class PDFViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        WebView mWebView = new WebView(PDFViewActivity.this);
        mWebView.getSettings().setJavaScriptEnabled(true);
        mWebView.getSettings().setPluginState(WebSettings.PluginState.ON);


        mWebView.loadUrl("https://docs.google.com/gview?embedded=true&url=" + "http://www.africau.edu/images/default/sample.pdf");
        setContentView(mWebView);

    }


}

File: app/src/main/java/com/muthu/pdfview/PDFViewActivity.kt
package com.muthu.pdfview

import android.annotation.SuppressLint
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.webkit.WebSettings
import android.webkit.WebView

@SuppressLint("SetJavaScriptEnabled")
class PDFViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val mWebView = WebView(this@PDFViewActivity)
        mWebView.settings.javaScriptEnabled = true
        mWebView.settings.pluginState = WebSettings.PluginState.ON


        mWebView.loadUrl("https://docs.google.com/gview?embedded=true&url=" + "http://www.africau.edu/images/default/sample.pdf")
        setContentView(mWebView)

    }


}


